### DISCUSSION TITLE
Should we have chat channel

### DISCUSSION THREAD

**Discussion by majkinetor** [1Remote/1Remote#407] [General]
Discord, Slack, or even custom like Mattermost on our infrastructure.

I am not so sure... I don't like that those things are ultimatelly unsearchable and conversations are effectivelly lost or hard to find.

I think GL discussions are enough and more user friendly. We can use [Q&A](https://github.com/1Remote/1Remote/discussions/categories/q-a) for concrete topics instead or General channel for more vague talks.

/cc @VShawn

**Comment by VShawn** [comment]
Chat channels are more suitable for follow-up discussions in real time or QA tasks you mentioned. However, the features provided by GitHub can basically meet these needs.

I personally rarely pay attention to chat channel. I prefer well-thought long-message, rather than thoughtless instant messaging.

So I think chat channels are not currently needed (I took a look at mRemoteNG's chat channel and there are very few messages).

**Reply by majkinetor** [reply]
OK, no chat then, I consider this matter resolved.
